moveit_ros/moveit_servo/include/moveit_servo/pose_tracking/pid_controller.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+namespace moveit_servo
+{
+class PIDController
```

Comment:
What's wrong with https://github.com/ros-controls/control_toolbox/blob/melodic-devel/include/control_toolbox/pid.h#L117
or even your own https://bitbucket.org/AndyZe/pid/src/8268b7250e40c33a08d7faa7706885eb317eadbd/include/pid/pid.h#lines-52 ?

Having yet another PID implementation in the ecosystem seems senseless, though you might have your reasons for doing this.

Reply:
Agree, I'll try to get the control_toolbox version working.

"My" PID package only works with ROS topics IIRC, but I want a fast C++ interface

Reply:
done